build.gradle

```diff
@@ -87,8 +87,10 @@ allprojects {
   }
 }
 
+/** Configure building for Java+Kotlin projects. */
 subprojects { project ->
   if (project.name == 'android-test') return
+  if (project.ext.artifactId == 'okhttp-bom') return
```

Comment:
Why the different mechanism than `name`?

Reply:
No good reason. Fixed to be consistent with above.